components/inputs/input-inline-help-mixin.js

```diff
@@ -0,0 +1,44 @@
+import { css, html } from 'lit';
+import { bodySmallStyles } from '../typography/styles.js';
+
+export const InputInlineHelpMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			_hasInlineHelp: { type: Boolean, reflect: true, attribute: '_has-inline-help' }
+		};
+	}
+
+	static get styles() {
+		const styles = [ bodySmallStyles, css`
+			:host([_has-inline-help]) .d2l-input-inline-help {
+				display: block;
+			}
+			.d2l-input-inline-help {
+				display: none;
+				margin-top: 0.5rem !important;
```

Comment:
Given that we potentially have cases where the inline-help could be limited in width, I think it would be worthwhile adding `overflow-wrap: anywhere;` to handle unbreakable text cases. 

In case you are curious, despite what it sounds like, [anywhere](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#anywhere) is actually pretty smart.